src/components/name-tag.js

```diff
@@ -198,31 +200,35 @@ AFRAME.registerComponent("name-tag", {
 
   updateFromPresenceMeta(presenceMeta) {
     this.displayName = presenceMeta.profile.displayName;
+    this.pronouns = presenceMeta.profile.pronouns;
     this.identityName = presenceMeta.profile.identityName;
     this.isRecording = !!(presenceMeta.streaming || presenceMeta.recording);
     this.isOwner = !!(presenceMeta.roles && presenceMeta.roles.owner);
     this.isTyping = !!presenceMeta.typing;
     this.isHandRaised = !!presenceMeta.hand_raised;
     if (this.isAvatarReady) {
       this.updateDisplayName();
+      this.updatePronouns();
       this.updateHandRaised();
       this.resizeNameTag();
     }
   },
 
+  updateNametagWidth(name) {
+    if (name) {
+      this.pronounsText?.el && this.pronounsText.el.components["text"].getSize(this.size);
```

Comment:
I think both `pronounText` and `nametagText` should exist at this point so we can probably remove all the defensive code from here. We can probably also remove the `name` param and just pass the method name to the listener
`this.nametagText.el.addEventListener("text-updated", this.updateNametagWidth, { once: true})`
as long as we bind it here:
https://github.com/mozilla/hubs/blob/79f2fc11836c4573c546fb980f12c83afd1fb0c4/src/components/name-tag.js#L64

Reply:
The check for `nametagText.el` was legacy code, but I removed and you're correct, works fine! 👍